### PR TITLE
Added swiper, like/dislike, add to favList and back buttons

### PR DESCRIPTION
**Pull request by juliaozm**

### CODE REVIEW

File: components/SwipePage.jsx [hunk 1]
```diff
@@ -1,76 +1,214 @@
...
+                justifyContent: "flex-start",
+              },
+            },
+          },
+        }}
+      />
+      <Buttons />
+    </View>
+  );
 };
```
Comment (Cam-Bloom) [inline components/SwipePage.jsx:161]: Think we should decide between all styles using stylesheet or inline, not both together

File: assets/utils/variables.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const colors = {
+	red: '#E94057',
+	green: '#27AE60',
+	white: '#fff',
+	darkgrey: '#574D4D',
+	border: '#E8E6EA',
+}
```
Comment (Cam-Bloom) [inline assets/utils/variables.js:7]: This is clean

**Review by JxckHind — APPROVED** [review]

**Review by Superjim — APPROVED** [review]

**Review by Cam-Bloom — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -9,6 +9,7 @@ npm-debug.*
 *.mobileprovision
 *.orig.*
 web-build/
+.prettierrc.js
```
Comment (Aleksanrda) [inline .gitignore:13]: Should .prettierrc file be stored in .gitignore?

Comment (Superjim) [reply .gitignore:13]: no

**Pull request merged**